README.md

```diff
@@ -223,20 +233,19 @@ targetManager:
 
 #### Loader
 
-The loader component of the `sparrow` will load the [Runtime](#runtime) configuration dynamically.
+The loader component of the `sparrow` will load the [checks](#checks) configuration dynamically during the runtime.
 
 The loader can be selected by specifying the `loaderType` configuration parameter.
 
-The default loader is an `http` loader that is able to get the runtime configuration from a remote endpoint.
+The default loader is an `http` loader that is able to get the checks configuration from a remote endpoint during the runtime.
 
 Available loader:
 
 - `http`: The default. Loads configuration from a remote endpoint. Token authentication is available. Additional
-  configuration parameters have the prefix `loaderHttp`.
-- `file` (experimental): Loads configuration once from a local file. Additional configuration parameters have the
-  prefix `loaderFile`. This is just for development purposes.
+  configuration parameters will be set in the `loader.http` section.
+- `file` (experimental): Loads configuration once from a local file. Additional configuration parameters will be set in the `loader.file` section. This is just for development purposes & is currently read just once. The target manager is not supported for this setup!
```

Comment:
What do you mean by "is not supported for this setup"? 

If you mean configuring the target manager on runtime, can you rephrase the sentence? It only confuses the user by thinking the target manager is configured on runtime.

Reply:
The reconciling is only triggered when a new config is sent to the config channel https://github.com/caas-team/sparrow/blob/c4a6b545c1b04d871c7e366f1604cc3d36bc1729/pkg/sparrow/run.go#L124 . We need to change that in the future.
Since the `file` loader is currently loading the local config file just once at startup the config never gets reconciled again. The target manager is not able to set the remote targets and will not be passed to the checks. We should change the `file` loader to load the config by an interval as well.

Reply:
I have updated the docu here... hopefully it is more clear now

Reply:
Thank you for clearing this up.

We should indeed add this feature to the file loader but imo it has minor priority.